src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
I can say for sure about CommonJs (browserify and webpack), where module scope is not a global scope so OpenSeadragon will not leak to the window object.
So this is not an issue there.

Correct me if I'm wrong, but to change it as you suggested one would need to change all source files, because they all expect OpenSeadragon variable in the scope.  And actually there is no actual need to do that.